### PR TITLE
Minor tweaks impacting usage

### PR DESCRIPTION
**Pull request by ChrisKeefe**
Closes https://github.com/qiime2/qiime2/issues/612
Closes https://github.com/qiime2/qiime2/issues/613

### CODE REVIEW

File: q2cli/core/usage.py [hunk 1]
```diff
@@ -134,11 +134,11 @@ def init_artifact(self, name, factory):
...
 
     def import_from_format(self, name, semantic_type,
-                           fmt_variable, view_type=None):
+                           variable, view_type=None):
         variable = super().import_from_format(
-            name, semantic_type, fmt_variable, view_type=view_type)
+            name, semantic_type, variable, view_type=view_type)
 
-        in_fp = fmt_variable.to_interface_name()
+        in_fp = variable.to_interface_name()
```
Comment (ebolyen) [inline q2cli/core/usage.py:141]: There's a namespace collision here with super(), I've seen this handled elsewhere by assigning the result to `var` and the parameter remains `variables`.

Comment (ChrisKeefe) [reply q2cli/core/usage.py:141]: Thanks for the catch. Why do we use two separate variables here, when we could just do all of the recording with the original `variable` arg, and then return the result of the `super().etc()` call at the end.

Is using `var` and `variable` about keeping us in line with the "sandwich pattern" mentioned in the dev docs so the method implementations feel similar? Are we concerned that the super method could side-affect this function's arguments? Are we trying to highlight the different-ness of the passed and super-generated UsageVariables?

Comment (thermokarst) [reply q2cli/core/usage.py:141]: I don't think we could do all of the recording with the original `variable` - these represent two completely different pieces of data. The `variable` is the raw format data, a `table.biom`, for example. The `var` is an `Artifact`, it represents the imported data.

`UsageVariable` is not a recorder - the `self.recorder` on this interface is responsible for recording state. Hope that helps.

File: q2cli/core/usage.py [hunk 2]
```diff
@@ -134,11 +134,11 @@ def init_artifact(self, name, factory):
...
-                           fmt_variable, view_type=None):
-        variable = super().import_from_format(
-            name, semantic_type, fmt_variable, view_type=view_type)
+                           variable, view_type=None):
+        var = super().import_from_format(
+            name, semantic_type, variable, view_type=view_type)
 
-        in_fp = fmt_variable.to_interface_name()
+        in_fp = variable.to_interface_name()
         out_fp = variable.to_interface_name()
```
Comment (ebolyen) [inline q2cli/core/usage.py:142]: This has to be changed to `var` (hopefully answering the question for why there are two). You can see these used on lines 153-4

**Review by ebolyen — CHANGES_REQUESTED** [review]

File: q2cli/core/usage.py (hunk 2 above)
Comment (ChrisKeefe) [reply q2cli/core/usage.py:142]: Sorry for all the derping around with this @ebolyen  and @thermokarst  - I've been making bad inferences from intermediate states when I should have been reading the original code. Happy New Year! :tada:

**Review by ebolyen — APPROVED** [review]
LGTM, ready for merge when tests pass. Thanks @ChrisKeefe!

**Comment by ebolyen** [comment]
oh they already did, sweet!

**Pull request merged**